docs/resources/anycast_config.md

```diff
@@ -3,14 +3,47 @@
 page_title: "bloxone_anycast_config Resource - terraform-provider-bloxone"
 subcategory: ""
 description: |-
-  RManages Anycast Configuration. Anycast configuration comprises common anycast configuration data that is defined in support of one service on a set of on-prem hosts.
+  Manages Anycast Configuration. Anycast configuration comprises common anycast configuration data that is defined in support of one service on a set of on-prem hosts.
 ---
 
 # bloxone_anycast_config (Resource)
 
-RManages Anycast Configuration. Anycast configuration comprises common anycast configuration data that is defined in support of one service on a set of on-prem hosts.
+Manages Anycast Configuration. Anycast configuration comprises common anycast configuration data that is defined in support of one service on a set of on-prem hosts.
 
+## Example Usage
 
+```terraform
+data "bloxone_infra_services" "anycast_services" {
+  filters = {
+    service_type = "anycast"
+  }
+}
+
+data "bloxone_infra_hosts" "anycast_hosts" {
+  filters = {
+    pool_id = data.bloxone_infra_services.anycast_services.results.0.pool_id
+  }
+}
+
+#Create anycast configuration with necessary fields
+resource "bloxone_anycast_ac_config" "example" {
```

Comment:
name hasn't been renamed in the example.

Reply:
done